Back-End/controllers/admin_controllers/students_controllers.js

```diff
@@ -1,6 +1,26 @@
 const db = require('../../db');
 
+
+const students_details = (req, res) => {
+    const query = `SELECT * FROM users WHERE role_id = 3`;
+    
+    db.query(query, (err, result) => {
+        if (err) throw err
+        res.json(result)
+    })
+}
+
+const disable_user = (req, res) => {
+    const query = `UPDATE users SET isDisabled = 1 WHERE id = ?`
+    const data = req.params.id 
```

Comment:
No semicolon, make sure to use prettier to format the code,  make sure to format all of the pages you have worked on

Reply:
Noted.